src/wallet/wallet.cpp

```diff
@@ -4184,8 +4184,8 @@ bool CWalletTx::AcceptToMemoryPool(const CAmount& nAbsurdFee, CValidationState&
     // We must set fInMempool here - while it will be re-set to true by the
     // entered-mempool callback, if we did not there would be a race where a
     // user could call sendmoney in a loop and hit spurious out of funds errors
-    // because we think that the transaction they just generated's change is
-    // unavailable as we're not yet aware its in mempool.
+    // because we think that the just generated transaction's change is
+    // unavailable as we're not yet aware it's in mempool.
```

Comment:
A bit unclear. Perhaps change it to "as we're not yet aware that it is in the mempool"?

Reply:
Fixed.